sql/core/src/main/scala/org/apache/spark/sql/parquet/ParquetTypes.scala

```diff
@@ -289,7 +295,20 @@ private[parquet] object ParquetTypesConverter extends Logging {
       name: String,
       nullable: Boolean = true,
       inArray: Boolean = false,
+      parquetSchema: Option[ParquetType] = None,
       toThriftSchemaNames: Boolean = false): ParquetType = {
+
+    val parquetElementTypeBySchema =
```

Comment:
Ah, yes, an early version had that, I somehow moved to this verbose code O.o

Thanks

Am 21. Juni 2015 02:33:15 MESZ, schrieb Davies Liu <notifications@github.com>:
>>        toThriftSchemaNames: Boolean = false): ParquetType = {
>> +
>> +    val parquetElementTypeBySchema =
>
>These could be
>```
>parquetSchema.filter(_.containsField(name)).map(_.getType(name))
>```
>
>---
>Reply to this email directly or view it on GitHub:
>https://github.com/apache/spark/pull/6796/files#r32889401

--